packages/firestore/test/lite/integration.test.ts

```diff
@@ -2128,6 +2168,28 @@ describe('countQuery()', () => {
     });
   });
 
+  it('count query with collection groups', () => {
+    return withTestDb(async db => {
+      const collectionGroupId = doc(collection(db, 'countTest')).id;
+      const docPaths = [
+        `${collectionGroupId}/cg-doc1`,
+        `abc/123/${collectionGroupId}/cg-doc2`,
+        `invalid${collectionGroupId}/cg-doc3`,
```

Comment:
nit: replace "invalid" with something that is obviously filler, like "zzzz". There is nothing "invalid" about having "invalid" in a collection name, so using the word "invalid" incorrectly implies that something is "invalid", which detracts from readabiltiy.